packages/css/src/components/badge/badge.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+.amsterdam-badge {
+  display: inline-block;
+  font-family: var(--amsterdam-badge-font-family);
+  font-size: var(--amsterdam-badge-spacious-font-size);
+  font-weight: var(--amsterdam-badge-font-weight);
+  line-height: var(--amsterdam-badge-spacious-line-height);
+  padding-inline: var(--amsterdam-badge-padding-inline);
+
+  .amsterdam-theme--compact & {
+    font-size: var(--amsterdam-badge-compact-font-size);
+    line-height: var(--amsterdam-badge-compact-line-height);
+  }
+}
+
+.amsterdam-badge--blue {
+  background-color: var(--amsterdam-badge-blue-background-color);
+  color: var(--amsterdam-badge-blue-color);
+}
+
+.amsterdam-badge--dark-blue {
+  background-color: var(--amsterdam-badge-dark-blue-background-color);
+  color: var(--amsterdam-badge-dark-blue-color);
```

Comment:
This should use the ‘primary blue’ colour.

It highlights an inconsistency in our colour names:
- Figma: ‘Light Blue’ (#009DE6) and ‘Blue’ (#004699)
- React: ‘Blue’ (#009DE6), ‘Primary Blue’ (#004699) and ‘Dark Blue’ (#102E62)

It appears we work around this in the `Spotlight` tokens with a slightly mischievous mapping from blue to dark blue etc.

We shouldn’t change too much in the names of our base colours in the scope of this PR, but we are defining a new API here. To prevent a breaking change next week already, let’s use ‘blue’ and ‘light blue’, and ‘green’ and ‘light green’ as prop values (Figma is also inconsistent between blue and green) and do the mapping in the tokens here as well.

I’d be happy to implement this if you like.

Reply:
Sure this confused me I must say, they use blue and light-blue, and we change it to blue = light-blue, dark-blue = blue. 

Perhaps we should not have (brand color) primary blue as a supplemental color to begin with and change our blue to light blue.

I also think that the background colours we make available should only consist of the supplemental colours and not add black and white just because we can.

Reply:
> I’d be happy to implement this if you like.

Please, be my guest! I've commented on colours a couple of times in Figma and do not fully follow what change is wrong or right.